workflow/custom-dashboard/dashboard-builder-overview.mdx

```diff
@@ -0,0 +1,194 @@
+---
+title: Dashboard Basics
+---
+
+The Dashboard Builder allows you to generate new dashboards in under a minute, integrate custom widgets into existing dashboards, or effortlessly clone default dashboards for further refinement. Drill into any data ported to Middleware with advanced filtering options by date range and metric value.
+
+# How Dashboards Work
+
+The Dashboard Builder is a Graphical User Interface (GUI) based approach that enables users to create complex query logic without the added stress of learning a new query language. The metrics made available to users when constructing dashboards are dynamically generated based on the data that users port to Middleware. For example, if you have instrumented Middleware's Infrastructure Agent and APM packages, you will have access to all of your host and application-level data. 
+
+<Note> For more information on instrumenting the Infra Agent and APM packages, navigate to the [Quickstart](https://docs.middleware.io/getting-started) page.</Note>
+
+# Create New Dashboard
+
+To get started, navigate to the Dashboard Builder page, select __Create Dashboard__, label your new dashboard and press __Save Changes__:
+
+![](https://lh7-us.googleusercontent.com/nAcWz6onmEDjro8MHLJW0QuAMxlrLqaoBsR--nCRpyRXrXOMnC6TItKPMU0JFcc8sUlhyYTqqtxFGCzwSuNuxDzlUrqfAephKP5kf_GXSKt9IwQ1ljclEQUhbnbw_Hov-Cvhu0B38ZbwBj1tU4U0Hes)
+
+![](https://lh7-us.googleusercontent.com/vnHYyys7-lF4JJ0iWNe1U_9tzKBaugYaHecw6-jK-QcyReb5eurxA7mRonIGf9mYI6KO0cVcbRB5i4WPy2DkILnpSglgxYzPllWNoUa-bX5YpWtvdY2M1rRDoaEaTQE7qwyqqkDRET3fX1bJsjoS-fY)
+
+<Tip> You can create a dashboard from an existing data source or create a widget in an existing dashboard. We recommend starting with a __New Dashboard__. </Tip>
+
+Create a new dashboard by selecting a data source to start your dashboard with:
+
+![](https://lh7-us.googleusercontent.com/awzI7eStfpqKWy78cOJdW-8pMyrVL6lrocPdQnmVlqrEtGh6QFjABrm-GoaCyG3URvGjdRnm_Ukzw4ZrbY6U-Li9QlY-eWKJ3dFzTzO_fb6IT48onLDEUZxaJ5EPDKJwWm6WLs_hzjmGJJTsCvq6AgQ)
+
+Select the metrics you would like to include as widgets in your dashboard:
+
+![](https://lh7-us.googleusercontent.com/MYBkwh9gWohgGbuF9CScQlUi5RU-0w0J8AVXTY3_MqEnidFnqTzGgXSLMy4iVcpV3_aRgrA3fCnxRZU53vGuL5Lp1RUL7ljVXmRdJfhbqptdDmcATY9LDiTDG715qWoX5ABfvTFuFVgmqArf6UC1i1I)
+
+Once you have added widgets to your dashboard, you will be taken directly to your dashboard:
+
+<Note> Data will appear momentarily after you have created your new dashboard. </Note>
+
+![](https://lh7-us.googleusercontent.com/rg4LfJetovyC5UMrgE7H3aN3UUuK0K2kbbHp5LYgGHLs12M-48jW3OtN7RlPA0sJnTznOpcdtNFqghehlroWY7lhsw1eQsE1ZSKXSqf8e-EUNX99RzcCr24ScSxFiXhp27NvLokBGwV9PMuKWk8jwTY)
+
+# Create New Widget
+
+Navigate to the dashboard you would like to add a new widget to and select __Add Widget__ and __Create Widget__:
```

Comment:
We should always try to intro sections with at least one more sentence. In this case since it's the second section of the doc we can say "Creating a fresh widget is useful for custom reporting or exploration. To create a new widget, navigate to the dashboard...."